tests/test_algorithm.py

```diff
@@ -4359,13 +4359,11 @@ def test_daily_delisted_equities(self,
             )
 
         # Check expected cash.
-        self.assertEqual(algo.cash, expected_cash)
```

Comment:
The way things worked previously is that we were doing the auto-closing before market open, so when the algo recorded its `cash` on the auto-close date, the cash had already been adjusted. Since the auto closing now happens at market close, when the algo records its `cash` on the auto close date, the cash adjustment hasn't happened yet. That means the `algo.cash` is essentially off-by-one day from the `'ending_cash'`. If you think it is worth it/not too confusing, I could keep this assertion here but shift the values by a day?